ui/src/app/pages/homepage/data-access-requirements.spec.tsx

```diff
@@ -931,5 +903,95 @@ describe('DataAccessRequirements', () => {
         wrapper = component();
         await waitOneTickAndUpdate(wrapper);
         expect(findModule(findControlledTierCard(wrapper), AccessModule.CTCOMPLIANCETRAINING).exists()).toBeFalsy();
-    });
+      });
+
+  it('Should display CT training when ineligible', async() => {
```

Comment:
please add a test for the other way to be CT-ineligible: there is no CT `tierEligibility` struct at all